pkg/api/datasources.go

```diff
@@ -63,7 +65,28 @@ func (hs *HTTPServer) GetDataSources(c *models.ReqContext) response.Response {
 	return response.JSON(200, &result)
 }
 
-func GetDataSourceById(c *models.ReqContext) response.Response {
+func (hs *HTTPServer) getDataSourceAccessControlMetadata(c *models.ReqContext, dsID int64) (accesscontrol.Metadata, error) {
+	if !hs.Cfg.FeatureToggles["accesscontrol"] || !c.QueryBool("metadata") {
```

Comment:
Would it be better to have accesscontrol, instead of metadata?
```suggestion
	if !hs.Cfg.FeatureToggles["accesscontrol"] || !c.QueryBool("accesscontrol") {
```

Reply:
I think metadata is too generic so i would go with accesscontrol

Reply:
Ok, I'll make the change :slightly_smiling_face: